### ISSUE TITLE
Bug - overall explanation edited text reverting to auto-text

### ISSUE THREAD

**Issue by MJWainwright** [epistemonikos/isoq#100]
i discovered a bug in the worksheet Evidence Profile.

- I completed each component assessment
- I completed the overall assessment and edited the minimum text provided
- I saved and the edited explanation appears
- HOWEVER, when I click edit the program is not displaying my edited explanation, it is displaying the original auto-text.

![image](https://user-images.githubusercontent.com/72507850/117716052-03810380-b1d1-11eb-91ba-a2b211d4f326.png)

But then I click Edit and what appears in the explanation box is the original auto-generated text (my edits are not visible).

![image](https://user-images.githubusercontent.com/72507850/117716228-3fb46400-b1d1-11eb-9fa8-8cd282a3eaf4.png)

**Comment by MJWainwright** [comment]
@damian-garrido I just checked on Production server. I only saw the fix after I cleared  my cache. Before clearing, I could not see the fix on an existing project. It seems like the fixes are being displayed on the test server without me having to clear my cache, but that is not the case on the production server. On the production server I still have to clear my cache. Is it possible the update you made to the test server regarding this was not applied to the production server?
